src/configs/directory.rs

```diff
@@ -9,7 +9,6 @@ pub struct DirectoryConfig<'a> {
     pub truncate_to_repo: bool,
     pub substitutions: IndexMap<String, &'a str>,
     pub fish_style_pwd_dir_length: i64,
-    pub use_logical_path: bool,
```

Comment:
Why did you remove this option?

Reply:
Apologies, I should have dropped a PR comment explaining this.

So I spent a bit of time considering this option and decided that it had the potential to be a bit of a footgun. As I understand it there's currently only one shell which supports logical paths (PowerShell) and I found that it was very easy to get "stuck" in a virtual path... here's an example of what I mean in GIF form:

![use_logical_path_false_behavior](https://user-images.githubusercontent.com/918952/104112236-23063d80-5351-11eb-876d-9b93997fc179.gif)

Reply:
I've restored this configuration flag, though I think clearing it (at least under PowerShell) would be a bad idea.